distributed/deploy/tests/test_cluster.py

```diff
@@ -43,3 +45,44 @@ def __init__(self):
 
     cluster = FooCluster()
     assert "foo" in cluster._cluster_info
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("asynchronous_cluster", [True, False])
+async def test_sync_defaults_to_cluster_setting(
```

Comment:
There is still something wrong. This test fails reliably on CI and locally for me with

```
=================================== FAILURES ===================================
_________________ test_sync_defaults_to_cluster_setting[True] __________________

asynchronous_cluster = True
loop_in_thread = <tornado.platform.asyncio.AsyncIOLoop object at 0x7f3e749a8a10>
cleanup = None

    @pytest.mark.asyncio
    @pytest.mark.parametrize("asynchronous_cluster", [True, False])
    async def test_sync_defaults_to_cluster_setting(
        asynchronous_cluster, loop_in_thread, cleanup
    ):
    
        cluster = Cluster(asynchronous=asynchronous_cluster)
        cluster.loop = loop_in_thread
    
        async def foo():
            return 1
    
        result = cluster.sync(foo)
    
        if asynchronous_cluster:
>           assert isinstance(result, CoroutineType)
E           assert False
E            +  where False = isinstance(1, CoroutineType)

distributed/deploy/tests/test_cluster.py:65: AssertionError
```

Reply:
Ah sorry this looks like an issue after resolving the merge conflict. I'll investigate and get back to you. Thanks for taking a look!